<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta name="description" content=""> <meta name="author" content=""> <title>VCaldas Personal website</title> <!-- Bootstrap core CSS --> <link href="/assets/css/main.css" rel="stylesheet"> <!-- Custom fonts for this template --> <!-- <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css"> --> <link href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css"> <link href='https://fonts.googleapis.com/css?family=Cabin:700' rel='stylesheet' type='text/css'> <!-- Custom styles for this template --> <!-- <link href="assets/styles/grayscale.css" rel="stylesheet"> --> <!-- Custom styles for this template --> <link href="/assets/css/particles.css" rel="stylesheet"> <!-- Temporary navbar container fix --> <style> .navbar-toggler { z-index: 1; } @media (max-width: 576px) { nav > .container { width: 100%; } } </style> </head> <body> <!-- Navigation --> <nav id="navcore" class="navbar navbar-core fixed-top navbar-toggleable-md navbar-light"> <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarExample" aria-controls="navbarExample" aria-expanded="false" aria-label="Toggle navigation"> Menu <i class="fa fa-bars"></i> </button> <div class="container"> <a class="navbar-brand" href="/"">{ V/Caldas }</a> <div class="collapse navbar-collapse" id="navbarExample"> <ul class="navbar-nav ml-auto"> <li class="nav-item"> <a class="nav-link" href="/publications">Publications</a> </li> <!-- <li class="nav-item"> <a class="nav-link" href="/blog">Blog</a> </li> --> <!-- <li class="nav-item"> <a class="nav-link" href="#projects">Projects</a> </li> <li class="nav-item"> <a class="nav-link" href="/contact.html">Contact</a> </li> --> <li class="nav-item"> <a class="nav-link" href="/readme">About</a> </li> </ul> </div> </div> </nav> <div class="container default-page-container"> <div class="row cv"> <div class="col col-md-8 col-md-offset-1"> <div class="container publications"> <h2 id="publications">Publications</h2> <h3 id="2017">2017</h3> <div class="publication" id="Aleklett2017"> <a name="Aleklett2017"></a> <b>Build your own soil: exploring microfluidics to create microbial habitat structure.</b> <br /> Kristin Aleklett, E. Toby Kiers, Pelle Ohlsson, Thomas S. Shimizu, Victor E. A. Caldas, Edith C. Hammer <br /><em>ISME Journal </em>, 2017. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="https://www.nature.com/articles/ismej2017184">URL</a>]</div> <div class="abstract"> In this perspectives paper, we show how microfluidics can be applied in soil microbial ecology, we present inspirational examples of microfluidic approaches in other fields such as biomedical cell research, and critically reflect on potential drawbacks and challenges to studying soil ecology with this technology.   </div><pre class="bibtex">@article{Aleklett2017}
</pre></div> <h3 id="2016">2016</h3> <div class="publication" id="Ghodke2016"> <a name="Ghodke2016"></a> <b>Single-molecule specific mislocalization of red fluorescent proteins in live <em>Escherichia coli</em></b>. <br /> Ghodke, H., Caldas, V. E. A., Punter, C. M., van Oijen, A. M., and Robinson, A. <br /><em>Biophysical Journal </em>, 2016. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://www.sciencedirect.com/science/article/pii/S0006349516303940">URL</a>]</div> <div class="abstract">Tagging of individual proteins with genetically encoded fluorescent proteins (FPs) has been used extensively to study localization and interactions in live cells. Recent developments in single-molecule localization microscopy have enabled the dynamic visualization of individual tagged proteins inside living cells. However, tagging proteins with FPs is not without problems: formation of insoluble aggregates and inhibition of native functions of the protein are well-known issues. Previously reported artifacts manifest themselves at all expression levels of the FP-tagged proteins, making the design of control experiments relatively straightforward. Here, we describe a previously uncharacterized mislocalization artifact of Entacmaea quadricolor red fluorescent protein variants that is detectable at the single-molecule level in live <em>Escherichia coli</em> cells. </div><pre class="bibtex">@article{Ghodke2016,
author = {Ghodke, Harshad and Caldas, Victor E A and Punter, Christiaan M. and van Oijen, Antoine M. and Robinson, Andrew},
doi = {10.1016/j.bpj.2016.05.047},
issn = {15420086},
journal = {Biophys. J.},
number = {1},
pages = {25--27},
pmid = {27338860},
title = {Single-Molecule Specific Mislocalization of Red Fluorescent Proteins in Live Escherichia coli},
volume = {111},
year = {2016}
}
</pre></div> <h3 id="2015">2015</h3> <div class="publication" id="Robinson2015"> <a name="Robinson2015"></a> <b>Regulation of mutagenic DNA polymerase V in <em>Escherichia coli</em> includes inner membrane binding and release</b>. <br /> A. Robinson, John P. McDonald, V.E.A. Caldas, M. Patel, E. A. Wood, C. M. Punter, H. Ghodke, M. M. Cox, R. Woodgate, M. F. Goodman, A. M. van Oijen. <br /> <em>PLOS Genetics</em>, 2015. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005482">URL</a>]</div> <div class="abstract">Spatial regulation is often encountered as a component of multi-tiered regulatory systems in eukaryotes, where processes are readily segregated by organelle boundaries. Well-characterized examples of spatial regulation are less common in bacteria. Low-fidelity DNA polymerase V (UmuD'2C) is produced in <em>Escherichia coli</em> as part of the bacterial SOS response to DNA damage. Due to the mutagenic potential of this enzyme, pol V activity is controlled by means of an elaborate regulatory system at transcriptional and posttranslational levels. Using single-molecule fluorescence microscopy to visualize UmuC inside living cells in space and time, we now show that pol V is also subject to a novel form of spatial regulation. After an initial delay ($\sim$ 45 min) post UV irradiation, UmuC is synthesized, but is not immediately activated. Instead, it is sequestered at the inner cell membrane. The release of UmuC into the cytosol requires the RecA* nucleoprotein filament-mediated cleavage of UmuD→UmuD'. Classic SOS damage response mutants either block [umuD(K97A)] or constitutively stimulate [recA(E38K)] UmuC release from the membrane. Foci of mutagenically active pol V Mut (UmuD'2C-RecA-ATP) formed in the cytosol after UV irradiation do not co-localize with pol III replisomes, suggesting a capacity to promote translesion DNA synthesis at lesions skipped over by DNA polymerase III. In effect, at least three molecular mechanisms limit the amount of time that pol V has to access DNA: (1) transcriptional and posttranslational regulation that initially keep the intracellular levels of pol V to a minimum; (2) spatial regulation via transient sequestration of UmuC at the membrane, which further delays pol V activation; and (3) the hydrolytic activity of a recently discovered pol V Mut ATPase function that limits active polymerase time on the chromosomal template. </div><pre class="bibtex">
@article{Robinson2015,
author = {Robinson, Andrew and McDonald, John P and Caldas, Victor E A and Patel, Meghna and Wood, Elizabeth A and Punter, Christiaan M and Ghodke, Harshad and Cox, Michael M and Woodgate, Roger and Goodman, Myron F and van Oijen, Antoine M},
doi = {10.1371/journal.pgen.1005482},
issn = {1553-7404},
journal = {PLoS Genet.},
month = {aug},
number = {8},
pages = {e1005482},
pmid = {26317348},
publisher = {Public Library of Science},
title = {Regulation of mutagenic DNA Polymerase V activation in space and time.},
volume = {11},
year = {2015}
}
</pre></div> <div class="publication" id="Caldas2015"> <a name="Caldas2015"></a> <b>iSBatch: a batch-processing platform for data analysis and exploration of live-cell single-molecule microscopy images and other hierarchical datasets</b>. <br /> V.E.A. Caldas, C.M. Punter, H. Ghodke, A. Robinson, A. M. van Oijen.<br /> <em>Molecular Biosystems </em>, 2015. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://pubs.rsc.org/is/content/articlelanding/2015/mb/c5mb00321k#!divAbstract">URL</a>]</div> <div class="abstract">Recent technical advances have made it possible to visualize single molecules inside live cells. Microscopes with single-molecule sensitivity enable the imaging of low-abundance proteins, allowing for a quantitative characterization of molecular properties. Such data sets contain information on a wide spectrum of important molecular properties, with different aspects highlighted in different imaging strategies. The time-lapsed acquisition of images provides information on protein dynamics over long time scales, giving insight into expression dynamics and localization properties. Rapid burst imaging reveals properties of individual molecules in real-time, informing on their diffusion characteristics, binding dynamics and stoichiometries within complexes. This richness of information, however, adds significant complexity to analysis protocols. In general, large datasets of images must be collected and processed in order to produce statistically robust results and identify rare events. More importantly, as live-cell single-molecule measurements remain on the cutting edge of imaging, few protocols for analysis have been established and thus analysis strategies often need to be explored for each individual scenario. Existing analysis packages are geared towards either single-cell imaging data or in vitro single-molecule data and typically operate with highly specific algorithms developed for particular situations. Our tool, iSBatch, instead allows users to exploit the inherent flexibility of the popular open-source package ImageJ, providing a hierarchical framework in which existing plugins or custom macros may be executed over entire datasets or portions thereof. This strategy affords users freedom to explore new analysis protocols within large imaging datasets, while maintaining hierarchical relationships between experiments, samples, fields of view, cells, and individual molecules. </div><pre class="bibtex">
@article{Caldas2015,
author = {Caldas, Victor E. A. and Punter, Christiaan M. and Ghodke, Harshad and Robinson, Andrew and van Oijen, Antoine M.},
doi = {10.1039/C5MB00321K},
issn = {1742-206X},
journal = {Mol. BioSyst.},
number = {10},
pages = {2699--2708},
pmid = {26198886},
title = {iSBatch: a batch-processing platform for data analysis and exploration of live-cell single-molecule microscopy images and other hierarchical datasets},
url = {http://xlink.rsc.org/?DOI=C5MB00321K},
volume = {11},
year = {2015}
}

</pre></div> <h3 id="2014">2014</h3> <div class="publication" id="Pozzi2014"> <a name="Pozzi2014"></a> <b>Microbial population in an aerated thermophilic reactor that treats recycled card-board plant wastewater</b>. <br /> Pozzi, E., Megda, C.R., Caldas, V. E. A., Damianovic, M., Pires, E. <br /> <em>J. Water Process Eng.</em> 2014. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://www.sciencedirect.com/science/article/pii/S2214714414000865">URL</a>]</div> <div class="abstract">Aerated thermophilic reactors may have advantages for treating industrial effluents discharged at high temperatures. In this study, the wastewater from a recycled packaging cardboard plant was treated at 55°C in a moving bed aerated reactor. It was observed that, although the overall dissolved oxygen was maintained at 2–3mgL−1, several anaerobic microorganisms are found either in the mixed liquor or in the attached biofilm. This result indicates that both aerobic and anaerobic mechanisms participate in pollutant removal in this reactor, which confers advantages when recalcitrant compounds are found. The median COD removal efficiency of 84% for 36h of HRT observed in this experiment is expected for non-optimized activated sludge treatment of high temperatures industrial wastewater. </div><pre class="bibtex">
@article{Pozzi2014,
author = {Pozzi, Eloisa and Megda, Claudia Regina and Caldas, Victor E.A. and Damianovic, M{\'{a}}rcia and Pires, Eduardo Cleto},
doi = {10.1016/j.jwpe.2014.08.011},
issn = {22147144},
journal = {J. Water Process Eng.},
keywords = {16S rRNA,Activated sludge,Anaerobic treatment,Clone library,Paper mill,Thermophilic wastewater treatment},
month = {dec},
pages = {74--81},
title = {Microbial population in an aerated thermophilic reactor that treats recycled cardboard plant wastewater},
url = {http://www.sciencedirect.com/science/article/pii/S2214714414000865},
volume = {4},
year = {2014}
}
</pre></div> <h3 id="2013">2013</h3> <div class="publication" id="DaSilva2013"> <a name="DaSilva2013"></a> <b>Selenocysteine Biosynthesis and Insertion Machinery in <em>Naegleria gruberi</em></b>. <br /> Da Silva, M.T.A., Caldas, V. E. A., Costa, F.C., Silvestre, D.A.M.M., Thiemann, O.H.<br /> <em>Molecular and Biochemical Parasitology - Cover</em> 2013. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://www.sciencedirect.com/science/article/pii/S0166685113000388">URL</a>]</div> <div class="abstract">Selenium (Se) is an essential trace element primarily found in selenoproteins as the 21st amino acid (selenocysteine, Sec, or U). Selenoproteins play an important role in growth and proliferation and are typically involved in cellular redox balance. Selenocysteine is encoded by an in-frame UGA codon specified by a stem-loop structure, the Sec insertion sequence element (SECIS), which, in eukaryotes, is located in the 3'-untranslated region (UTR). The availability of the Naegleria gruberi (ATCC 30224) genome sequence and the use of this organism as a model system for the pathogenic amoeba N. fowleri allowed us to investigate the Sec incorporation pathway in this primitive eukaryote. Using bioinformatics tools, we identified gene sequences encoding PSTK (O-phosphoseryl-tRNA(Sec) kinase), SepSecS (O-phosphoseryl-tRNA:selenocysteinyl-tRNA synthase), SelD/SPS2 (selenophosphate synthetase), EFSec (selenocysteine-specific elongation factor) and SBP (SECIS binding protein). These findings were confirmed by RT-PCR and by sequencing. A potential tRNA(Ser)Sec (SelC) gene and a putative selenoprotein with sequence similarity to a mitochondrial thioredoxin reductase (TR3) were also identified. Our results show that the selenocysteine incorporation machinery is indeed present in N. gruberi. Interestingly, the SelD/SPS2 gene is 2214 bp in length and contains two distinct domains. The N-terminal region shows sequence similarity to predicted methyltransferase proteins, and the C-terminal region is homologous to prokaryotic SelD/SPS2. Our results suggest the possibility of novel selenoproteins. </div><pre class="bibtex">
@article{DaSilva2013,
author = {da Silva, M T A and Caldas, V E A and Costa, F C and Silvestre, D A M M and Thiemann, O H},
doi = {10.1016/j.molbiopara.2013.04.002},
file = {:Users/caldas/Dropbox/Science/Papers/Mendeley/da Silva et al. - 2013 - Molecular and biochemical parasitology - Selenocysteine biosynthesis and insertion machinery in Naegleria grube.pdf:pdf},
issn = {1872-9428},
journal = {Mol. Biochem. Parasitol.},
keywords = {Amino Acid Sequence,Base Sequence,Biosynthetic Pathways,Biosynthetic Pathways: genetics,Computational Biology,DNA,Gene Expression Profiling,Genes,Models,Molecular,Molecular Sequence Data,Naegleria,Naegleria: genetics,Naegleria: metabolism,Nucleic Acid Conformation,Protozoan,Reverse Transcriptase Polymerase Chain Reaction,Selenocysteine,Selenocysteine: biosynthesis,Selenoproteins,Selenoproteins: metabolism,Sequence Alignment,Sequence Analysis},
month = {apr},
number = {2},
pages = {87--90},
pmid = {23603359},
title = {Selenocysteine biosynthesis and insertion machinery in Naegleria gruberi.},
url = {http://www.sciencedirect.com/science/article/pii/S0166685113000388},
volume = {188},
year = {2013}
}
</pre></div> <div class="publication" id="DelNery2013"> <a name="DelNery2013"></a> <b>Long-term performance and operational strategies of a poultry slaughterhouse waste stabilization pond system in a tropical climate</b>. <br /> Del Nery, V., Damianovic, M.H.Z., Pozzi, E., De Nardi, I.R., Caldas, V. E. A., Pires, E.C. <br /> <em>Resources, Conservation and Recycling</em>, 2013. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://www.sciencedirect.com/science/article/pii/S0921344912002054">URL</a>]</div> <div class="abstract">This paper discusses the performance of a poultry slaughterhouse wastewater treatment plant (WWTP) in a tropical country and operating strategies that were applied over 84 months to this WWTP. The WWTP consisted of a dissolved air flotation (DAF) system, anaerobic ponds (APs) and a facultative pond (FP). The organic loading rates applied to the APs ranged from 0.26 to 1.05kgBOD/m3day, and the hydraulic retention time ranged from 2.2 to 3.8days. The resultant COD and BOD removal efficiencies were 42±11% and 38±15%, respectively. The FP surface loading was up to 983kgBOD/haday. The main observed events in the WWTP processes over this period were organic overload, low chemical oxygen demand (COD) and biochemical oxygen demand (BOD) removal efficiencies and color change from green to reddish at the FP. Molecular biology techniques were used to identify microbial groups in the FP in the period of low organic matter removal efficiency and when the liquid became reddish. Sunny conditions favored anoxygenic photosynthesis, the growth of purple nonsulfur bacteria (such as Rhodocyclales-like organisms), and green nonsulfur bacteria (such as Chloroflexi-like organisms). Mechanical aeration at the inlet of the FP restored the water to its original color and increased the COD and BOD removal efficiencies. Based on the BOD removal efficiencies of 89%, after the installation of aerators, the operating strategies illustrate the WWTP system's capacity and versatility to adapt to load variations. </div><pre class="bibtex">
@article{DelNery2013,
abstract = {},
author = {Del Nery, V. and Damianovic, M.H.Z. and Pozzi, E. and de Nardi, I.R. and Caldas, V.E.A. and Pires, E.C.},
doi = {10.1016/j.resconrec.2012.11.006},
issn = {09213449},
journal = {Resour. Conserv. Recycl.},
keywords = {Anaerobic pond,Facultative pond,Full-scale,Phylogenetic tree,Poultry slaughterhouse wastewater},
month = {feb},
pages = {7--14},
title = {Long-term performance and operational strategies of a poultry slaughterhouse waste stabilization pond system in a tropical climate},
url = {http://www.sciencedirect.com/science/article/pii/S0921344912002054},
volume = {71},
year = {2013}
}
</pre></div> <h3 id="2011">2011</h3> <div class="publication" id="Serrao2011"> <a name="Serrao2011"></a> <b>Promiscuous interactions of human septins: the GTP binding domain of SEPT7 forms filaments within the crystal</b>. <br /> Serrão, V.H., Alessandro, F., Caldas, V. E. A., Marcal, R.L., Pereira, H.D., Thiemann, O.H., Garrat, R.C. <br /> <em>FEBS Letters</em>, 2011. <div class="pubLinks">[<a href="#" class="abstractLink">Abstract</a> | <a href="#" class="bibtexLink">BibTeX</a> | <a href="http://www.sciencedirect.com/science/article/pii/S0014579311007940">URL</a>]</div> <div class="abstract">We describe the purification, crystallization and structure for the GTP-binding domain of human septin 7 (SEPT7G). We show that it forms filaments within the crystal lattice which employ both the G and NC interfaces, similar to those seen in the hetero-filament of SEPT2/6/7. The NC interface is considered promiscuous as it is absent from the hetero-filament. Such promiscuity could provide the potential for permuting monomers along a filament in order to generate diversity in hetero-polymers. On the other hand, our results suggest that the G and NC interfaces may be necessary but insufficient for determining correct hetero-filament assembly. </div><pre class="bibtex">
@article{Serrao2011,
author = {Serr{\~{a}}o, Vitor Hugo Balasco and Alessandro, Fernando and Caldas, Victor Emanoel Armini and Mar{\c{c}}al, Rafaela Leite and Pereira, Humberto D'Muniz and Thiemann, Otavio Henrique and Garratt, Richard Charles},
doi = {10.1016/j.febslet.2011.10.043},
file = {:Users/caldas/Dropbox/Science/Papers/Mendeley/Serr{\~{a}}o et al. - 2011 - FEBS letters - Promiscuous interactions of human septins the GTP binding domain of SEPT7 forms filaments within.pdf:pdf},
issn = {1873-3468},
journal = {FEBS Lett.},
keywords = {Cell Cycle Proteins,Cell Cycle Proteins: chemistry,Cell Cycle Proteins: isolation &amp; purification,Cell Cycle Proteins: metabolism,Crystallography, X-Ray,Guanosine Triphosphate,Guanosine Triphosphate: metabolism,Humans,Models, Molecular,Protein Structure, Tertiary,Protein Subunits,Protein Subunits: chemistry,Protein Subunits: metabolism,Septins,Septins: chemistry,Septins: isolation &amp; purification,Septins: metabolism,Substrate Specificity},
month = {dec},
number = {24},
pages = {3868--73},
pmid = {22064074},
title = {Promiscuous interactions of human septins: the GTP binding domain of SEPT7 forms filaments within the crystal.},
url = {http://www.sciencedirect.com/science/article/pii/S0014579311007940},
volume = {585},
year = {2011}
}
</pre></div> <h2 id="theses">Theses</h2> <p><strong>PhD Thesis.</strong> Caldas, V.E.A. <a href="http://www.rug.nl/research/portal/en/publications/activation-and-regulation-of-e-coli-dna-polymerase-v-studied-at-the-singlemolecule-level%28b84ae014-e8c0-4d24-ad6e-95efd4711d3b%29.html">Activation and regulation of E. coli DNA Polymerase V studied at the single-molecule level</a>. Zernike Institute PhD Thesis series. University of Groningen. The Latex template is available <a href="https://github.com/vcaldas/thesisTemplate">here.</a></p> <p><strong>M.Sc. Thesis.</strong> Caldas, V.E.A. <a href="http://www.teses.usp.br/teses/disponiveis/76/76132/tde-19102011-141742/pt-br.php">Adenine phosphoribosyltransferase from Schistosoma mansoni: insights into the catalytic mechanism via molecular dynamics</a>. Physics Institute at São Carlos - University of São Paulo.</p> </div> </div> <div class="col col-md-4" style="border-left: 1px solid #eee8d5;"><!-- sidebar --> <h4>Profiles and Metrics</h4> <ul class="author__urls social-icons"> <li><a href="mailto:v.arminicaldas@vu.nl"><i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> v.arminicaldas@vu.nl</a></li> <li><a href="https://orcid.org/0000-0002-4263-0786"><i class="ai ai-orcid" aria-hidden="true"></i> ORCID</a></li> <li><a href="http://buscatextual.cnpq.br/buscatextual/visualizacv.do?id=K4254688H5"><myicon class="icon"><img alt="Lattes Icon" src="/img/lattes-fav-icon.png"></myicon> Lattes CV</a></li> <li><a href="https://scholar.google.com/citations?user=vdKIs70AAAAJ&hl=en"><i class="ai ai-google-scholar-square" aria-hidden="true"></i> Google Scholar</a></li> <li><a href="https://nl.linkedin.com/in/vcaldas"><i class="fa fa-fw fa-linkedin-square" aria-hidden="true"></i> LinkedIn</a></li> <li><a href="https://github.com/vcaldas"><i class="fa fa-fw fa-github" aria-hidden="true"></i> Github</a></li> <li><a href="/cv.pdf"><i class="fa fa-user-circle-o " aria-hidden="true"></i> CV</a></li> </ul> </div> </div> </div> </body> <footer class="footer"> <div class="text-center"> <p>Build with Jekyll and <span class="love">❤</span>. Get the source<a href="https://github.com/vcaldas/vcaldas.github.io"> here!</a> </div> </footer> <!-- javascript --> <!-- Bootstrap core JavaScript --> <script src="/assets/js/jquery.js"></script> <script src="/assets/js/tether.js"></script> <script src="/assets/js/bootstrap.js"></script> <!-- Plugin JavaScript --> <script src="/assets/js/jquery.easing.js"></script> <!-- Custom scripts for this template --> <script src="/assets/js/grayscale.js"></script> <script src="/assets/js/particles.js"></script> <script src="/assets/js/sweet-scroll.js"></script> <script src="/assets/js/particle.js"></script> <script src="../assets/js/pubs.js"></script> </html>
